cmd/node/config/economics.toml

```diff
@@ -13,20 +13,20 @@
 
 [FeeSettings]
     MaxGasLimitPerBlock = "1500000000"
-    MinGasPrice = "100000000000000" #will yield min tx fee of 10ERD
+    MinGasPrice = "200000000000" #will yield min tx fee of 0.01ERD
```

Comment:
This means 0.002ERD applying DataLimitForBaseCalc 10^4

Reply:
MinGasPrice * MinGasLimit - gaslimit of a transaction is 50.000